skflow/estimators/dnn.py

```diff
@@ -17,48 +17,18 @@
 
 from sklearn.base import ClassifierMixin, RegressorMixin
 
-from skflow.estimators.base import TensorFlowEstimator
+from skflow.estimators.base import TensorFlowEstimator, ESTIMATOR_COMMON_DOCSTRING
 from skflow import models
+from skflow.util.doc_utils import Appender
 
 
 class TensorFlowDNNClassifier(TensorFlowEstimator, ClassifierMixin):
-    """TensorFlow DNN Classifier model.
```

Comment:
I'm a bit torn about this... I understand why you are trying to do this, but, for example, sklearn has for all it's the estimators a doc string explicitly - like here https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tree.py and it kinda make sense when you are just browsing code.